maestro-cli/src/main/java/maestro/cli/device/DeviceService.kt

```diff
@@ -0,0 +1,199 @@
+package maestro.cli.device
+
+import com.github.michaelbull.result.get
+import dadb.Dadb
+import io.grpc.ManagedChannelBuilder
+import ios.idb.IdbIOSDevice
+import maestro.MaestroTimer
+import maestro.cli.CliError
+import maestro.cli.device.ios.Simctl
+import maestro.cli.util.EnvUtils
+import java.io.File
+
+object DeviceService {
+
+    fun startDevice(device: Device.AvailableForLaunch): Device.Connected {
+        when (device.platform) {
+            Platform.IOS -> {
+                Simctl.launchSimulator(device.modelId)
+                Simctl.awaitLaunch(device.modelId)
+                return Device.Connected(
+                    instanceId = device.modelId,
+                    description = device.description,
+                    platform = device.platform,
+                )
+            }
+            Platform.ANDROID -> {
+                val androidHome = EnvUtils.androidHome()
+                    ?: throw CliError("ANDROID_HOME environment variable is not set")
+
+                val emulatorBinary = File(androidHome, "emulator/emulator")
+                    .takeIf { it.exists() }
+                    ?: File(androidHome, "tools/emulator")
```

Comment:
This should be `emulator/emulator` I believe

Reply:
See line 30. I use this as a fallback